api/geth_backend.go

```diff
@@ -783,6 +783,30 @@ func (b *GethStatusBackend) AppStateChange(state string) {
 	b.log.Info("App State changed", "new-state", s)
 	b.appState = s
 
+	if s == appStateForeground {
+		wallet, err := b.statusNode.WalletService()
+		if err != nil {
+			return
```

Comment:
We should log this with `log.Error` because it's a big deal. I am even wondering whether we should bubble this error to status-react through signals.